src/constraints.jl

```diff
@@ -655,7 +655,7 @@ into account in the primal value of the constraint. For instance, the constraint
 evaluation of `2x + 3y`.
 ```
 """
-function value(con_ref::ConstraintRef{Model, <:_MOICON}; result::Int = 1)
+function value(con_ref::ConstraintRef{<:AbstractModel, <:_MOICON}; result::Int = 1)
```

Comment:
It doesn't really make sense to make this an `AbstractModel`, unless we also change the signature on `_constraint_primal`.